.github/workflows/ci.yml

```diff
@@ -17,10 +17,10 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10, 3.11]
         install_variant: ["pip"]
         include:
-        - python-version: 3.10.8
+        - python-version: 3.11
```

Comment:
Why this change? I would say to just add supported versions the above list should be enough. The following is just additional included variants.

Reply:
Ok, So additions should be just the supported versions. Thanks.

Reply:
The mix
```
          python-version: 3.10.8
          install_variant: "rpm.fedora.34"
```
must remain the same so you should revert your change here. The reason is that this defines a "test variant" that only works on Fedora 34 OS which always has exactly python 3.10.8 installed there.

Reply:
Ok, it seems you also got this right now, resolving.